bentoml/adapters/base_input.py

```diff
@@ -91,7 +91,7 @@ def parse_cli_input(cli_args: Iterable[str]) -> Iterator[bytes]:
     input_g.add_argument('--input', nargs="+", type=str)
     input_g.add_argument('--input-file', nargs="+")
 
-    parsed_args, _ = parser.parse_known_args(cli_args)
+    parsed_args, _ = parser.parse_known_args(list(cli_args))
```

Comment:
@akainth015 why list here?

Reply:
The function signature takes an Optional Sequence, but the variable is an iterator. Maybe the issue is with the function signature.